src/Symfony/Component/Process/PhpProcess.php

```diff
@@ -55,6 +61,9 @@ public function __construct($script, $cwd = null, array $env = null, $timeout =
      */
     public function setPhpBinary($php)
     {
+        if ('\\' !== DIRECTORY_SEPARATOR) {
+            $php = 'exec '.$php;
```

Comment:
i'm going to remove this one because it breaks if anyone already prefix $php with exec